File: C++/Iniciante/1020.cpp
#include <iostream>
using namespace std;
  
int main(){
    int C, D, E, A;
    
    cin >> A;
    
    C=A/365;
    D=A%365/30;
    E=A%365%30;
    
    cout << C << " ano(s)" << endl;
    cout << D << " mes(es)" << endl;
    cout << E << " dia(s)" << endl;
	return(0);
}

File: Python3/Iniciante/1020.py
# -*- encoding: utf-8 -*-

N = int(input())

ANOS = int(N/365)
N = N - (ANOS * 365)
MESES = int(N/30)
DIAS = N - (MESES * 30)

print("%d ano(s)" % ANOS)
print("%d mes(es)" % MESES)
print("%d dia(s)" % DIAS)
